gazebo/rendering/WideAngleCamera.cc

```diff
@@ -467,6 +471,13 @@ void WideAngleCamera::Load()
 
     if (sdfLens->HasElement("env_texture_size"))
       this->dataPtr->envTextureSize = sdfLens->Get<int>("env_texture_size");
+
+    const std::string envTextureFormat = "gazebo:env_texture_format";
```

Comment:
cc @chapulina @iche033 should we change this xml namespace to `ignition:` to match the mecum wheel parameter namespace?

Reply:
I asked them on slack, and they recommend changing the xml namespace prefix from `gazebo` to `ignition`

we also need to declare the xml namespace; I'll add an inline comment on the world file